src/signals/settings/roles/containers/RoleFormContainer/components/RoleForm/index.js

```diff
@@ -21,6 +21,7 @@ const StyledInput = styled(Input)`
 
 export const RoleForm = ({ role, permissions, onPatchRole, onSaveRole }) => {
   const [name, setName] = useState('');
+  const [isSubmitting, setSubmitting] = useState(false);
```

Comment:
Why not use the `useFormValidation` hook?

Reply:
hoe? ik zie het niet in de instellingen branch: wel de definitie, maar niet de toepassing in het userform?

Reply:
You can find the hook in `src/hooks/useFormValidation`. See #494 for implementation details.

Reply:
can you help me with that on Monday?

Reply:
gedaan